openhumans/models.py

```
@@ -95,6 +95,21 @@ def _refresh_tokens(self, client_id, client_secret):
             self.token_expires = self.get_expiration(data['expires_in'])
             self.save()
 
+    def delete_single_file(self, file_id=None, file_basename=None):
```

I'm sorry if what I wrote previously was unclear. This function should raise an error if it receives neither `file_id` nor `file_basename`. The function should _also_ raise an error if it receives _both_ `file_id` and `file_basename`. These errors should be raised in this function, because...

While `ohapi` will also raise an error for these situations, the content of this error message isn't appropriate to the function as implemented here, because it refers to an `all_files` parameter not used here. https://github.com/OpenHumans/open-humans-api/blob/master/ohapi/api.py#L184)

Thanks! :)